lib/strava/models/mixins/start_date_local.rb

```diff
@@ -0,0 +1,27 @@
+module Strava
+  module Models
+    module Mixins
+      module StartDateLocal
+        extend ActiveSupport::Concern
+
+        included do
+          property 'start_date_local', transform_with: ->(v) { ::Time.parse(v) }
+          def start_date_local
+            shift = begin
+                if timezone.include?('+')
+                  "+0#{timezone.split('+').last.to_i}:00"
+                elsif timezone.include?('-')
+                  "-0#{timezone.split('-').last.to_i}:00"
+                end
+                    rescue NameError
+                      "-0#{((start_date - self['start_date_local']) / 3600).to_i}:00"
+              end
+
+            local_start_date = self['start_date_local']
```

Comment:
Doing `self['start_date_local']` twice in this code, pluck it out above into a variable.

Nit: `local_start_date` and `start_date_local` are too similar, I would just make a two-letter variable `dt` ;)